stack/corepersistence/queryindex/src/main/java/org/apache/usergrid/persistence/index/impl/EsEntityIndexBatchImpl.java

```diff
@@ -255,7 +247,7 @@ private static Map entityToMap( EntityObject entity ) {
 
             if ( f instanceof ListField ) {
                 List list = ( List ) field.getValue();
-                entityMap.put( field.getName().toLowerCase(),
```

Comment:
Should we use the same prefix for arrays and lists?  They have the same json representation, so it seems we'll want them to map the same in ES.